bot/exts/info/information.py

```diff
@@ -42,15 +43,33 @@ def get_channel_type_counts(guild: Guild) -> DefaultDict[str, int]:
         return channel_counter
 
     @staticmethod
-    def get_member_counts(guild: Guild) -> Dict[str, int]:
+    def join_role_stats(role_ids: list[int], name: str, guild: Guild) -> dict[str, int]:
+        """Return a dictionary with the number of `members` of each role given, and the `name` for this joined group."""
+        members = []
+        for role_id in role_ids:
+            if (role := guild.get_role(role_id)) is not None:
+                members.append(role.members)
+            else:
+                raise NonExistentRoleError(role_id)
+        return {name: len(set(members))}
+
+    @staticmethod
+    def get_member_counts(guild: Guild) -> dict[str, int]:
         """Return the total number of members for certain roles in `guild`."""
-        roles = (
-            guild.get_role(role_id) for role_id in (
-                constants.Roles.helpers, constants.Roles.mod_team, constants.Roles.admins,
-                constants.Roles.owners, constants.Roles.contributors,
-            )
+        role_ids = [constants.Roles.helpers, constants.Roles.mod_team, constants.Roles.admins,
+                    constants.Roles.owners, constants.Roles.contributors]
+        roles = []
+        for role_id in role_ids:
+            if (role := guild.get_role(role_id)) is not None:
+                roles.append(role)
+            else:
+                raise NonExistentRoleError(role_id)
+
+        role_stats = {role.name.title(): len(role.members) for role in roles}
+        role_stats.update(
+            **Information.join_role_stats([constants.Roles.project_leads, constants.Roles.domain_leads], "Leads", guild)
```

Comment:
I actually don't remember if this unpacking is necessary.

Additionally, you can reach this staticmethod through `self`.

Reply:
`get_member_counts` is a staticmethod as well.